### PR TITLE
Linux warn upd

### PR DESCRIPTION
**Pull request by yaute74**
Hello,
My proposal for the warning when update via menu after install via package.

PS: add also files in .gitignore for vs code IDE.

### CODE REVIEW

File: src/main/resources/sprache/English.properties [hunk 1]
```diff
@@ -857,6 +857,7 @@ updateDEVavailable=New Development Version available:
 updateBETAavailable=New Beta Version available:
 updateStableavailable=New Version available:
 ls.button.update.available=Update can be downloaded from:
+ls.button.update.warning.pkg=You install HO via a package.\nFor the coherence of your system it is preferable to continue to update the application by the same way.\nDo you still want to continue
```
Comment (akasolace) [inline src/main/resources/sprache/English.properties]: ls.button.update.linux.pkg.warning

Comment (yaute74) [reply src/main/resources/sprache/English.properties]: ok

**Comment by yaute74** [comment]
modification done

**Comment by akasolace** [comment]
@yaute74 
I cannot try it at the moment but it looks good to me and I trust you tried it yourself.
Thanks a lot for incorporating all the changes so  quickly.
nice PR !!

**Pull request merged**